### PR TITLE
[Frontend] Integrate Get Team on Admin Panel Fixed

### LINKED ISSUE #789: [Frontend] Integrate Get Team on Admin Panel
Deliverables-
1. Under Admin Panel, `About Us` => `Manage Team` It should call the Get Team api and the actual data should be populated
2. Clean the redundant team data from the test folder

Note - This issue is dependent on https://github.com/HITK-TECH-Community/Community-Website/issues/723

### PR DESCRIPTION
**Pull request by Hemu21**
## Issue that this pull request solves

Under Admin Panel, About Us => Manage Team It should call the Get Team api and the actual data should be populated

## Issue 
[Issue Link](https://github.com/HITK-TECH-Community/Community-Website/issues/789) resolve #789 

 Closes: #789

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Important information

There a important thing I am telling. Db does not had valid profiles so dp is not displaying. and Api get all team members get data without authenticating. I had added the token in fetching so no problem in merging this pr. But You need to Improve backend api.

### CODE REVIEW

File: frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx [hunk 1]
```diff
@@ -1,23 +1,58 @@
...
-              <h4> {d.name} </h4>
+            <div style={{display:"flex", justifyContent:"center",alignItems:"center"}}>
+              <img src={d.image} style={{margin:"auto", borderRadius:"50%", width:"200px", height:"200px"}} alt="" />
             </div>
+            <h4> {d.full_name} </h4>
             <div className={style["content"]}>
-              {d.description}
+              <h3 style={{fontWeight:"bolder"}}>About</h3> {d.description}
+            </div>
+            <div className={style["content"]}>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx:45]: the linkedIn, github and twitter urls should be url and should open in a new tab once someone clicks on it.

File: frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx [hunk 2]
```diff
@@ -1,23 +1,58 @@
...
           <div
             className={style["crd"]}
-            key={d.name}
+            key={d.full_name}
           >
-            <div className={style["head1"]}>
-              {d.icon}
-              <h4> {d.name} </h4>
+            <div style={{display:"flex", justifyContent:"center",alignItems:"center"}}>
+              <img src={d.image} style={{margin:"auto", borderRadius:"50%", width:"200px", height:"200px"}} alt="" />
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx:39]: If the image doesn't load due to some reason, can we show a default image?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
minor comments and please make sure the responsivenes as well, rest the pr looks good

File: frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx (hunk 2 above)
Comment (Hemu21) [reply frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx:39]: ok I will make this changes

**Comment by Hemu21** [comment]
@Kajol-Kumari  I had updated it can you please check. I am putting this image here because database image urls are wrong. so image is not displaying.

kajol

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**